spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/CompositePropagationFactoryTests.java

```diff
@@ -86,6 +93,24 @@ void inject() {
 			assertThat(request).containsOnly(entry("a", "a-value"), entry("b", "b-value"));
 		}
 
+		@Test
+		void thisIsATest() {
+			this.contextRunner
+					.withPropertyValues("management.tracing.propagation.type=B3",
+							"management.tracing.brave.span-joining-supported=true")
+					.run((context) -> {
+						CompositePropagationFactory factory = new CompositePropagationFactory(List.of(field("a"), field("b")),
+								List.of(field("c")));
+						Propagation<String> propagation = factory.get();
+						Tracing tracing = context.getBean(Tracing.class);
+						Span parentSpan = tracing.tracer().nextSpan();
+						Span childSpan = tracing.tracer().joinSpan(parentSpan.context());
+						Map<String, String> request = new HashMap<>();
+						propagation.injector(new CompositePropagationFactoryTests.MapSetter()).inject(parentSpan.context(), request);
```

Comment:
This test does not contain any verification steps: it will ~never fail, thus it is not very useful. The `Map` should contain the B3 header (with the parent id).
Also, please check the docs of Micrometer Tracing how to create a child span (you need to open a scope).